docs/site/Relations.md

```diff
@@ -0,0 +1,44 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relations.html
+summary:
+---
+
+## Overview
+
+Individual models are easy to understand and work with. But in reality, models
+are often connected or related. When you build a real-world application with
+multiple models, you’ll typically need to define relations between models. For
+example:
+
+- A customer has many orders and each order is owned by a customer.
+- A user can be assigned to one or more roles and a role can have zero or more
+  users.
+- A physician takes care of many patients through appointments. A patient can
+  see many physicians too.
+
+With connected models, LoopBack exposes as a set of APIs to interact with each
+of the model instances and query and filter the information based on the
+client’s needs.
+
+Model relation in LoopBack 3 is one of its powerful features which help users
+define real-world mappings between their models, access sensible CRUD APIs for
+each of the models, and add querying and filtering capabilities for the relation
+APIs after scaffolding their LoopBack applications. In LoopBack 4, with the
+introduction of [repositories](Repositories.md), we aim to simplify the approach
+to relations by creating constrained repositories. This means that, based on the
+relation definition, certain constraints need to be honoured by the target model
+repository, and thus, we produce a constrained version of it as a navigational
+property on the source repository.
+
+Here are the currently supported relations:
+
+- [HasMany](HasMany-relation.md)
+
+The articles on each type of relation above will show you how to leverage the
```

Comment:
I wonder if we should provide an example of what the controller route of a related model would look like here. I think it'd provide users with good idea of what `relation` involves in code

Reply:
Well we provide all that information at the `hasMany` relation page, so wouldn't that be sufficient?

Reply:
Sure, but I think that readers shouldn't be expected to find an example of `relation` in general being used in the pages dedicated to each of the relations.

That being said, probably not a big deal